index.js

```diff
@@ -144,7 +145,14 @@ function cli(process, beforeMinifyCallback) {
 
   // ... and do the magic!
   if (program.args.length > 0) {
-    minify(process, options, configurations, expandGlobs(program.args));
+    if (inputOptions.watch) {
+      var inputPaths = expandGlobs(program.args).map(function (path) { return path.expanded; });
```

Comment:
Since we use `expandGlobs(program.args)` three times here, should we extract it into a variable?

Reply:
I've extracted the call to expandGlobs() into a variable "expandedGlobs"